lib/transform.js

```diff
@@ -3,7 +3,8 @@
 var _             = require('underscore')
   , anchor        = require('anchor-markdown-header')
   , updateSection = require('update-section')
-  , getHtmlHeaders = require('./get-html-headers');
+  , getHtmlHeaders = require('./get-html-headers')
+  , md            = require('markdown-to-ast');
```

Comment:
I saw line numbers are used to order headers, and `markdown-to-ast` gives location info. I search on Google and Github for "markdown parsers" and "markdown ast" and used `markdown-to-ast` because it was simpler to use. I did see other options; `marked-ast` doesn't give locations. As for `markdown-ast`... should I try it?